### ISSUE TITLE
Reason for penalty loss for BC agent

### ISSUE THREAD

**Issue by gunshi** [takuseno/d3rlpy#188]
Hey,
Thanks for maintaining this library! Was wondering if you could point me to the reasoning/source for using this penalty loss over the logits for behavior cloning?
Link: [https://github.com/takuseno/d3rlpy/blob/803919b4df18515a5334226d499e6b655a32022d/d3rlpy/models/torch/imitators.py#L162]

Best,
Gunshi

**Comment by takuseno** [comment]
Let me close this issue. You can freely reopen this foe further discussion.
